website2/src/app/clean-air-forum/sessions/page.tsx

```
@@ -0,0 +1,163 @@
+
+  return (
+    <div className="bg-gray-100 rounded-lg shadow-sm p-4 mb-4">
+      <div
+        className="flex justify-between items-center cursor-pointer"
+        onClick={onToggle}
+      >
+        <div>
+          <h2 className="text-lg font-bold">{title}</h2>
+          <div
+            className="text-sm text-gray-600"
+            dangerouslySetInnerHTML={{ __html: renderContent(subText) }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)